plugins/image-prioritizer/tests/test-helper.php

```diff
@@ -235,7 +256,12 @@ public function data_provider_test_filter_tag_walker_visitors(): array {
 						</head>
 						<body>
 							<img data-od-added-fetchpriority data-od-removed-loading="lazy" fetchpriority="high" src="https://example.com/foo.jpg" alt="Foo" width="1200" height="800"  srcset="https://example.com/foo-480w.jpg 480w, https://example.com/foo-800w.jpg 800w" sizes="(max-width: 600px) 480px, 800px" crossorigin="anonymous">
-							<img data-od-removed-fetchpriority="high" src="https://example.com/bar.jpg" alt="Bar" width="10" height="10" loading="lazy" >
+							<p>Pretend this is a super long paragraph that pushes the next image mostly out of the initial viewport.</p>
+							<img data-od-removed-fetchpriority="high" data-od-removed-loading="lazy" src="https://example.com/bar.jpg" alt="Bar" width="10" height="10"  >
+							<p>Now the following image is definitely outside the initial viewport.</p>
+							<img data-od-added-loading data-od-removed-fetchpriority="high" loading="lazy" src="https://example.com/baz.jpg" alt="Baz" width="10" height="10" >
+							<img data-od-removed-fetchpriority="high" data-od-replaced-loading="eager" src="https://example.com/qux.jpg" alt="Qux" width="10" height="10"  loading="lazy">
+							<img src="https://example.com/quux.jpg" alt="Quux" width="10" height="10" loading="lazy"><!-- This one is all good. -->
```

Comment:
This is the key test showing how the new functionality is optimizing lazy-loading. Compare this markup with the input markup above.